src/core/messaging/ACSClient.ts

```diff
@@ -280,6 +280,13 @@ export class ACSConversation {
 
         try {
             const listener = (event: TypingIndicatorReceivedEvent) => {
+                const {sender, recipient} = event;
+
+                // Ignore participant's own typing events
+                if ((sender as any).communicationUserId === (recipient as any).communicationUserId) { // eslint-disable-line @typescript-eslint/no-explicit-any

```

Comment:
+1 for null check. Also, can we add telemetry if these information is not present and is required for the operation.

Reply:
It's a ACS typing event and should always contain a sender & receipient with its id.